test/benchmarks/driverBench/common.js

```diff
@@ -24,7 +24,9 @@ function loadSpecString(filePath) {
 }
 
 function makeClient() {
-  this.client = new MongoClient(process.env.MONGODB_URI || 'mongodb://127.0.0.1:27017');
+  this.client = new MongoClient(process.env.MONGODB_URI || 'mongodb://127.0.0.1:27017', {
+    timeoutMS: 0
```

Comment:
Let's merge without this. We want our benchmarks to run in default mode. we'll merge more bench tasks that change this setting

Reply:
Do we have a ticket tracking creating more benchmarks? (I assume we want the tasks that you have on your perf test branch)

Reply:
I think [NODE-6459](https://jira.mongodb.org/browse/NODE-6459) covers the tracking we need. Here we can just revert this setting.